plugin/web/extensions/sidebar/modals/welcomescreen/index.tsx

```diff
@@ -0,0 +1,191 @@
+import welcomeScreenVideo from "@web/extensions/sidebar/core/assets/welcomeScreenVideo.png";
+// import Backdrop from "@web/extensions/sidebar/modals/welcomescreen/component/Backdrop";
```

Comment:
Should be using this Backdrop to wrap everything, right?

Reply:
removed - based on discussion with Red

Reply:
e8d02a5faeea4dc4732056ccaf88ca6f433e776c